src/falco/config.vsh.yaml

```diff
@@ -0,0 +1,197 @@
+name: falco
+description: A C++ drop-in replacement of FastQC to assess the quality of sequence read data
+keywords: [qc, fastqc, sequencing]
+links:
+  documentation: https://falco.readthedocs.io/en/latest/
+  repository: https://github.com/smithlabcode/falco
+references:
+  doi: 10.12688/f1000research.21142.2
+license: GPL-3.0
+requirements:
+  commands: [falco]
+
+# Notes:
+# - falco as arguments similar to -subsample and we update those to --subsample
+# - The outdir argument is not required
+# - The input argument in falco is positional but we changed this to --input
+argument_groups:
+  - name: Input arguments
+    arguments:
+      - name: --input
+        required: true
+        type: file
+        multiple: true
+        description: input fastq files
+        example: input1.fastq;input2.fastq
+
+  - name: Run arguments
+    arguments:
+      - name: --nogroup
+        type: boolean_true
+        description: |
+          Disable grouping of bases for reads >50bp. 
+          All reports will show data for every base in 
+          the read. WARNING: When using this option, 
+          your plots may end up a ridiculous size. You 
+          have been warned!
+      - name: --contaminents
+        type: file
+        description: |
+          Specifies a non-default file which contains 
+          the list of contaminants to screen 
+          overrepresented sequences against. The file 
+          must contain sets of named contaminants in 
+          the form name[tab]sequence. Lines prefixed 
+          with a hash will be ignored. Default: 
+          /tmp/falco-1.2.2/Configuration/contaminant_list.txt
+      - name: --adapters
+        type: file
+        description: |
+          Specifies a non-default file which contains 
+          the list of adapter sequences which will be 
+          explicity searched against the library. The 
+          file must contain sets of named adapters in 
+          the form name[tab]sequence. Lines prefixed 
+          with a hash will be ignored. Default: 
+          /tmp/falco-1.2.2/Configuration/adapter_list.txt
+      - name: --limits
+        type: file
+        description: |
+          Specifies a non-default file which contains 
+          a set of criteria which will be used to 
+          determine the warn/error limits for the 
+          various modules. This file can also be used 
+          to selectively remove some modules from the 
+          output all together. The format needs to 
+          mirror the default limits.txt file found in 
+          the Configuration folder. Default: 
+          /tmp/falco-1.2.2/Configuration/limits.txt
+      - name: -subsample
+        alternatives: [-s]
+        type: boolean_true
```

Comment:
The description seems to suggest that this should be an integer? (must be a multiple of the provided value)

Reply:
True, my mistake!